assertj-core/src/main/java/org/assertj/core/error/AbstractAssert_satisfiesAllOf_Test.java

```diff
@@ -0,0 +1,33 @@
+package org.assertj.core.error;
+
+import org.assertj.core.api.Condition;
+import org.assertj.core.condition.AllOf;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.BDDAssertions.then;
+
+public class AbstractAssert_satisfiesAllOf_Test {
```

Comment:
Could you refactor that test so that it follows our contribution guidelines https://github.com/assertj/assertj/blob/3.x/CONTRIBUTING.md, especially the use of GIVEN WHEN THEN and `AssertionUtil.expectAssertionError`